### DISCUSSION TITLE
Write reflectivity values

### DISCUSSION THREAD

**Discussion by stefano73-ilmeteo** [ARM-DOE/pyart#1463] [Q&A] [answered]
Hi everyone,
does pyart allow to write reflectivity values into a netcdf file?
Thanks.

Stefano

**Comment by mgrover1** [comment, marked as answer]
Thanks for the question @stefano73-ilmeteo - you can use the `pyart.io.write_cfradial` function to accomplish this task!

https://arm-doe.github.io/pyart/API/generated/pyart.io.write_cfradial.html#pyart.io.write_cfradial

**Reply by stefano73-ilmeteo** [reply]
@mgrover1 is there any example showing how to write the reflectivtiy value?

**Reply by mgrover1** [reply]
There is an example writing out a cfradial file https://arm-doe.github.io/pyart/notebooks/changing_fields_and_saving.html
